EIPS/eip-7500.md

```diff
@@ -0,0 +1,190 @@
+---
+eip: 7500
+title: NFT Dynamic Traits
+description: Extension to ERC-721 and ERC-1155 for dynamic onchain traits
+author: Adam Montgomery (@montasaurus), Ryan Ghods (@ryanio), 0age (@0age)
+discussions-to: https://ethereum-magicians.org/t/erc-7501-nft-redeemables/15485
```

Comment:
discussion is here: https://ethereum-magicians.org/t/erc-7500-nft-dynamic-traits/15484

Reply:
thanks! fixed